From b33f11e87f38525b3cec2cf27e93d7efa50650fb Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Wed, 26 Aug 2015 23:40:38 +0800
Subject: [PATCH] MLK-11426: fix mfgtool kernel build error

Fixed build error:

drivers/built-in.o: In function `usb_charger_is_present':
:(.text+0x16d998): undefined reference to `power_supply_changed'
drivers/built-in.o: In function `ci_hdrc_imx_remove':
:(.text+0x16f924): undefined reference to `power_supply_unregister'
drivers/built-in.o: In function `ci_hdrc_imx_probe':
:(.text+0x170094): undefined reference to `power_supply_register'
:(.text+0x170144): undefined reference to `power_supply_unregister

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 arch/arm/configs/imx_v7_mfg_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_mfg_defconfig b/arch/arm/configs/imx_v7_mfg_defconfig
index 174e6a8..2e4c0fb 100644
--- a/arch/arm/configs/imx_v7_mfg_defconfig
+++ b/arch/arm/configs/imx_v7_mfg_defconfig
@@ -160,6 +160,7 @@ CONFIG_I2C_IMX=y
 CONFIG_SPI=y
 CONFIG_SPI_IMX=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_POWER_SUPPLY=y
 # CONFIG_HWMON is not set
 CONFIG_THERMAL=y
 CONFIG_CPU_THERMAL=y
-- 
1.8.0

